Cabal/doc/getting-started.rst

```diff
@@ -164,6 +164,6 @@ What Next?
 ----------
 
 Now that you know how to set up a simple Haskell package using Cabal, check out
-some of thee resources on the Haskell website's `documentation page
+some of these resources on the Haskell website's `documentation page
```

Comment:
I think it should be "the", otherwise it doesn't make much sense...

Reply:
```suggestion
some of the resources on the Haskell website's `documentation page
```